go/client/logclient.go

```diff
@@ -111,11 +121,11 @@ func New(uri string) *LogClient {
 // representation of the structure in |res|.
 // Returns a non-nil |error| if there was a problem.
 func (c *LogClient) fetchAndParse(uri string, res interface{}) error {
-	req, _ := http.NewRequest("GET", uri, nil)
-	resp, err := c.httpClient.Do(req)
+	resp, err := c.httpClient.Get(uri)
 	if err != nil {
 		return err
 	}
+	defer resp.Body.Close()
```

Comment:
Just because I'd then have to interpose it between the `ReadAll` and the subsequent `if err...` checking which made me sad.